ol3_viewer_prepare.sh

```diff
@@ -1,6 +1,23 @@
 #!/bin/bash
 
-echo "preparing ol3-viewer"
-rm  -rf libs
-mkdir -p libs
-ant
+compile_viewer=0
+if [ "$#" -gt 0 ] && [ "$1" = "DEV" ]; then
+    if [ -f "libs/ol3-viewer.js" ]; then
+        timestamp=$(date +%s)
+        echo $timestamp
+        timestamp_viewer=$(stat -f%c libs/ol3-viewer.js)
```

Comment:
After discussion with @waxenegger, 
if the file is present in "dev" mode, we won't check the time.
Will add a note to the readme (other PR already opened)